.env

```diff
@@ -1,6 +1,8 @@
 # node
 NODE_ENV=development
 
+HOST=localhost # change me to 0.0.0.0 for remote access
```

Comment:
We could add `# HOST=127.0.0.1` per default into .env.local

Reply:
do you mean `# HOST=0.0.0.0`?

Reply:
Of course 😅 But this is just an idea, can be done later.